src/main/java/com/dqu/simplerauth/managers/ConfigManager.java

```diff
@@ -4,13 +4,15 @@
 import com.google.common.io.Files;
 import com.google.gson.Gson;
 import com.google.gson.GsonBuilder;
+import com.google.gson.JsonArray;
 import com.google.gson.JsonObject;
 import net.fabricmc.loader.api.FabricLoader;
 
 import java.io.BufferedReader;
 import java.io.BufferedWriter;
 import java.io.File;
 import java.nio.charset.StandardCharsets;
+import java.util.Locale;
 
 public class ConfigManager {
     public static final int VERSION = 1;
```

Comment:
Should be version 2, and update the config to the new format if it is version 1.